moped-database/views/project_list_view.sql

```diff
@@ -284,11 +284,11 @@ LEFT JOIN min_estimated_phase_dates mepd ON mp.project_id = mepd.project_id
 LEFT JOIN project_component_work_types pcwt ON mp.project_id = pcwt.project_id
 LEFT JOIN LATERAL (
     SELECT
-        mpn.project_note,
-        mpn.created_at AS date_created
-    FROM moped_proj_notes mpn
-    WHERE mpn.project_id = mp.project_id AND mpn.project_note_type = 2 AND mpn.is_deleted = false
-    ORDER BY mpn.created_at DESC
+        combined_project_notes.project_note,
+        combined_project_notes.created_at AS date_created
+    FROM combined_project_notes
+    WHERE (combined_project_notes.project_id = mp.project_id OR mp.should_sync_ecapris_statuses = true AND mp.ecapris_subproject_id IS NOT null AND combined_project_notes.ecapris_subproject_id = mp.ecapris_subproject_id) AND (combined_project_notes.note_type_name = any(ARRAY['Status Update'::text, 'eCapris Status Update'::text])) AND combined_project_notes.is_deleted = false
```

Comment:
why is it `OR mp.should_sync_ecapris_statuses` and not `AND` ?

Reply:
This would be another great place for a comment if we could get them to stick between the bot processing. This is saying that we want to select from Moped status updates OR (if eCAPRIS syncing is enabled and a project has an eCAPRIS subproject ID) eCAPRIS status updates.

I'm also noticing that the parentheses that I used to group the logic here didn't stick around when the Postgres-created view was read into this file. That makes this harder to follow imo. I'll double check that the view is following what I described above.

Reply:
thank you for helping me parse it ❤️